src/Components/test/E2ETest/Tests/ComponentRenderingTestBase.cs

```diff
@@ -458,7 +456,7 @@ public void CanUseFocusExtensionToFocusElementPreventScroll()
         string getFocusedElementId() => Browser.SwitchTo().ActiveElement().GetAttribute("id");
 
         // A local helper that gets window.PageYOffset
-        long getPageYOffset() => (long)((IJavaScriptExecutor)Browser).ExecuteScript("return Math.round(window.pageYOffset)");
+        int getPageYOffset() => Convert.ToInt32(((IJavaScriptExecutor)Browser).ExecuteScript("return window.pageYOffset"));
```

Comment:
As an unrelated fix, this E2E test was failing for me locally even though it somehow passes in CI, perhaps because the screen size/DPI in CI means it happens to get an int value for the scroll position whereas I get a fractional value locally. 

This update makes it pass locally too.

Reply:
I've seen this happen to me locally too